src/app/components/static-footer/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import autobind from 'autobind-decorator';
+
+import styles from './styles.css';
+import image from './giphy.gif';
+
+// const cx = classNames.bind(styles);
```

Comment:
Oh, just me here with super important review comments like that this commented out code probably isn't necessary.

![everyday hero](https://media2.giphy.com/media/3f1gJUjkhmqwU/giphy.gif?fingerprint=e1bb72ff59a2fafd595952793663ac3a)

Reply:
Crap.